### PR TITLE
homework-9

### PR DESCRIPTION
**Pull request by DenisKishylia**
Doing 9 homework. Adding "Homes guests loves" mobile and computer versions. And fixing minor previous mistakes

### CODE REVIEW

File: Triphouse-project/style.css [hunk 1]
```diff
@@ -115,6 +131,7 @@
   font-size: 50px;
   line-height: 60px;
   margin-bottom: 100px;
+  padding-right: 650px;
```
Comment (BubentsovKirill) [inline Triphouse-project/style.css:134]: 2021-04-26_21h04_02
What is sens ?

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -151,6 +151,54 @@ <h2>What do we offer</h2>
...
+                </li>
+                <li>
+                    <img src="./img/rec2_mob.svg" alt=""/>
+                    <a href="#">Apartment Sunshine</a>
+                    <p>Santa  Cruz de Tenerife, Spain</p>
+                </li>
+            </ul>    
+        </div>
+    </section>
     <footer>
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:202]: 2021-04-26_21h01_43
Deferent contet width.

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
                     <img src="./ikons/logo.svg" alt="logo" />
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:17]: 2021-04-26_21h04_02
2021-04-26_21h01_43


Check paddings, width, and other properties for font-size.

File: Triphouse-project/style.css (hunk 1 above)
Comment (DenisKishylia) [reply Triphouse-project/style.css:134]: fix

Comment (DenisKishylia) [reply Triphouse-project/style.css:134]: fix

File: Triphouse-project/index.html (hunk 3 above)
Comment (DenisKishylia) [reply Triphouse-project/index.html:17]: done

**Pull request merged**